gio/sys/src/lib.rs

```diff
@@ -17313,7 +17321,21 @@ extern "C" {
     pub fn g_unix_mount_is_system_internal(mount_entry: *mut GUnixMountEntry) -> gboolean;
     pub fn g_unix_mount_points_changed_since(time: u64) -> gboolean;
     pub fn g_unix_mount_points_get(time_read: *mut u64) -> *mut glib::GList;
+    #[cfg(feature = "v2_82")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v2_82")))]
+    pub fn g_unix_mount_points_get_from_file(
```

Comment:
This needs to be conditional for non-Windows

Reply:
Hm we don't do that for any others so it probably doesn't matter after all

Reply:
Shouldn't those be in the gio-unix gi file nowadays anyway?

Reply:
I think that was not done for backwards compatibility reasons